x-pack/test/functional_execution_context/tests/browser.ts

```diff
@@ -12,7 +12,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'dashboard', 'header', 'home']);
   const retry = getService('retry');
 
-  describe('Browser apps', () => {
+  // Failing: See https://github.com/elastic/kibana/issues/112102
+  describe.skip('Browser apps', () => {
```

Comment:
With the new additional context, can we update the tests?

Reply:
The tests are already up to day.
I tried reenabling them, but they still randomly fail